src/components/editor/Store.tsx

```diff
@@ -22,6 +36,11 @@ export interface EditorStoreState<T> {
     serverUpdate: any | null;
     setServerUpdate: (newVal: EditorStoreState<T>['serverUpdate']) => void;
 
+    isSaving: boolean;
+    isEditing: boolean;
```

Comment:
These are helpful and right now adding computed values to Zustand is a pain... and honestly just setting these down below when needed was so much easier.